ocs_ci/ocs/warp.py

```diff
@@ -118,24 +160,46 @@ def run_benchmark(
         self.concurrent = concurrent if concurrent else self.concurrent["concurrent"]
         self.objects = objects if objects else self.objects["objects"]
         self.obj_size = obj_size if obj_size else self.obj_size["obj.size"]
-        self.pod_obj.exec_cmd_on_pod(
-            f"{self.warp_bin_dir} mixed "
-            f"--duration={self.duration} "
-            f"--host={self.host} "
-            f"--access-key={self.access_key} "
-            f"--secret-key={self.secret_key} "
-            f"--noclear --noprefix --concurrent={self.concurrent} "
-            f"--objects={self.objects} "
-            f"--obj.size={self.obj_size} "
-            f"--get-distrib={self.get_distrib} "
-            f"--put-distrib={self.put_distrib} "
-            f"--delete-distrib={self.delete_distrib} "
-            f"--stat-distrib={self.stat_distrib} "
-            f"--bucket={self.bucket_name} "
-            f"--analyze.out={self.output_file}",
-            out_yaml_format=False,
-            timeout=timeout,
-        )
+        self.client_str = ""
+        for client in self.client_ips:
+            self.client_str += f"{self.client_ips[client]}:7761,"
+        self.client_str = self.client_str.rstrip(",")
+        if multi_client:
+            self.pod_obj.exec_cmd_on_pod(
+                f"{self.warp_bin_dir} mixed "
+                f"--duration={self.duration} "
+                f"--host={self.host} "
+                f"--insecure={insecure} "
+                f"--tls={tls} "
+                f"--debug={debug} "
+                f"--access-key={self.access_key} "
+                f"--secret-key={self.secret_key} "
+                f"--obj.size={self.obj_size} "
+                f"--bucket={self.bucket_name} "
+                f"--warp-client={self.client_str} "
+                f"--analyze.out={self.output_file}",
+                out_yaml_format=False,
+                timeout=timeout,
+            )
+        else:
+            self.pod_obj.exec_cmd_on_pod(
+                f"{self.warp_bin_dir} mixed "
+                f"--duration={self.duration} "
+                f"--host={self.host} "
+                f"--access-key={self.access_key} "
+                f"--secret-key={self.secret_key} "
+                f"--noclear --noprefix --concurrent={self.concurrent} "
+                f"--objects={self.objects} "
+                f"--obj.size={self.obj_size} "
+                f"--get-distrib={self.get_distrib} "
+                f"--put-distrib={self.put_distrib} "
+                f"--delete-distrib={self.delete_distrib} "
+                f"--stat-distrib={self.stat_distrib} "
+                f"--bucket={self.bucket_name} "
+                f"--analyze.out={self.output_file}",
+                out_yaml_format=False,
+                timeout=timeout,
+            )
```

Comment:
I think that conditionally adding the options for the multi_client case is cleaner. Something like:

```
base_options = "".join(
    f"{self.warp_bin_dir} mixed ",
    f"--duration={self.duration} ",
    f"--host={self.host} ",
    f"--access-key={self.access_key} ",
    f"--secret-key={self.secret_key} ",
    f"--noclear --noprefix --concurrent={self.concurrent} ",
    f"--objects={self.objects} ",
    f"--obj.size={self.obj_size} ",
    f"--get-distrib={self.get_distrib} ",
    f"--put-distrib={self.put_distrib} ",
    f"--delete-distrib={self.delete_distrib} ",
    f"--stat-distrib={self.stat_distrib} ",
    f"--bucket={self.bucket_name} ",
    f"--analyze.out={self.output_file}",
)

multi_client_options = "".join(
    f"--insecure={insecure} "
    f"--tls={tls} "
    f"--debug={debug} "
)

cmd = f"{self.warp_bin_dir} mixed " + base_options + multi_client_options if multi_client else ""

self.pod_obj.exec_cmd_on_pod(cmd, out_yaml_format=False, timeout=timeout)
```